src/components/OfframpSummaryDialog/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { ArrowDownIcon } from '@heroicons/react/20/solid';
+import { FC } from 'preact/compat';
+import Big from 'big.js';
+
+import { InputTokenDetails, OutputTokenDetails } from '../../constants/tokenConfig';
+import { UseTokenOutAmountResult } from '../../hooks/nabla/useTokenAmountOut';
+import { useGetAssetIcon } from '../../hooks/useGetAssetIcon';
+import { useOfframpFees } from '../../hooks/useOfframpFees';
+import { useNetwork } from '../../contexts/network';
+import { Networks } from '../../helpers/networks';
+
+import { ExchangeRate } from '../ExchangeRate';
+import { NetworkIcon } from '../NetworkIcon';
+import { Dialog } from '../Dialog';
+
+interface AssetDisplayProps {
+  amount: string;
+  symbol: string;
+  iconSrc: string;
+  iconAlt: string;
+}
+
+const AssetDisplay = ({ amount, symbol, iconSrc, iconAlt }: AssetDisplayProps) => (
+  <div className="flex items-center justify-between w-full">
+    <span className="text-lg font-bold">
+      {amount} {symbol}
+    </span>
+    <img src={iconSrc} alt={iconAlt} className="w-8 h-8" />
+  </div>
+);
+
+interface FeeDetailsProps {
+  network: Networks;
+  feesCost: string;
+  fiatSymbol: string;
+  tokenOutAmount: UseTokenOutAmountResult;
+  fromToken: InputTokenDetails;
+  anchorUrl?: string;
+}
+
+const FeeDetails = ({ network, feesCost, fiatSymbol, tokenOutAmount, fromToken, anchorUrl }: FeeDetailsProps) => (
+  <section className="mt-6">
+    <div className="flex justify-between mb-2">
+      <p>Offramp fees</p>
+      <p className="flex items-center gap-2">
+        <NetworkIcon network={network} className="w-4 h-4" />
+        <strong>
+          {feesCost} {fiatSymbol}
+        </strong>
+      </p>
+    </div>
+    <div className="flex justify-between mb-2">
+      <p>Quote</p>
+      <p>
+        <strong>
+          <ExchangeRate tokenOutData={tokenOutAmount} fromToken={fromToken} toTokenSymbol={fiatSymbol} />
+          ($1.00)
```

Comment:
While it's in the reference image, I think we should remove this. As far as I understand, the idea of this '($1.00)' would be to indicate the dollar value of the resulting quote. This means, in order to make this value make sense, we would need to fetch the ARS or EUR to USD exchange rate from somewhere else and compare our output token amount to that. I think this is too complex and we should rather remove it for now. If this becomes a hard requirement later, we can still add it in the future. 
```suggestion
```